testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1109,3 +1109,11 @@
 Then(/^I should see the text "(.*?)" in the (Operating System|Architecture|Channel Label) field/) do |text, field|
   page.has_field?(text, with: field)
 end
+
+Then(/^I should see the current time as (Start Time|End Time)/) do |timestamp|
+  now = Time.now
+  #page.has_field?(now.strftime('%k:%M'), with: timestamp)
+  page.all(:xpath, '//div[@class="table table-striped vertical-middle"]').each do |td|
+    next unless td.has_selector?("#{now.between?(now-5, now+5)}")
```

Comment:
🤭 This will try to find the text "True" inside the 'td' webElement

Reply:
You're right this method didn't make much sense